@navikt/core/react/src/overlays/action-menu/ActionMenu.tsx

```diff
@@ -0,0 +1,1106 @@
+   * @example
+   * ```jsx
+   * <ActionMenu.RadioGroup
+   *   onValueChange={handleValueChange}
+   *   value={radioValue}
+   *   label="Radio group"
+   * >
+   *   <ActionMenu.RadioItem value="1">Radio 1</ActionMenu.RadioItem>
+   *   <ActionMenu.RadioItem value="2">Radio 2</ActionMenu.RadioItem>
+   * </ActionMenu.RadioGroup>
+   * ```
+   */
+  RadioGroup: typeof ActionMenuRadioGroup;
+  /**
+   * A radio item in the menu. Should always be grouped with a `ActionMenu.RadioGroup`.
+   * @example
+   * ```jsx
+   * <ActionMenu.RadioGroup
+   *   onValueChange={handleValueChange}
+   *   value={radioValue}
+   *   label="Radio group"
+   * >
+   *   <ActionMenu.RadioItem value="1">Radio 1</ActionMenu.RadioItem>
+   *   <ActionMenu.RadioItem value="2">Radio 2</ActionMenu.RadioItem>
+   * </ActionMenu.RadioGroup>
+   * ```
+   */
+  RadioItem: typeof ActionMenuRadioItem;
+  /**
+   * A simple divider to separate items in the menu.
+   */
+  Divider: typeof ActionMenuDivider;
+  /**
+   * A sub-menu that can be nested inside the menu.
+   * The sub-menu can be nested inside other sub-menus allowing for multiple levels of nesting.
+   * @example
+   * ```jsx
+   * <ActionMenu.Sub>
+   *   <ActionMenu.SubTrigger>Submenu 1</ActionMenu.SubTrigger>
+   *   <ActionMenu.SubContent>
+   *     <ActionMenu.Item>
+   *       Subitem 1
+   *     </ActionMenu.Item>
+   *     <ActionMenu.Item>
+   *       Subitem 2
+   *     </ActionMenu.Item>
+   *   </ActionMenu.SubContent>
+   * </ActionMenu.Sub>
+   * ```
+   */
+  Sub: typeof ActionMenuSub;
+  /**
+   * Acts as a trigger for a sub-menu.
+   * In contrast to `ActionMenu.Trigger`, this trigger is a standalone component and should not be wrapped around a React.ReactNode.
+   * @example
+   * ```jsx
+   * <ActionMenu.Sub>
+   *   <ActionMenu.SubTrigger>Submenu 1</ActionMenu.SubTrigger>
+   * </ActionMenu.Sub>
+   * ```
+   */
+  SubTrigger: typeof ActionMenuSubTrigger;
+  /**
+   * The content of a sub-menu.
+   * @example
+   * ```jsx
+   * <ActionMenu.Sub>
+   *   <ActionMenu.SubContent>
+   *     <ActionMenu.Item>
+   *       Subitem 1
+   *     </ActionMenu.Item>
+   *     <ActionMenu.Item>
+   *       Subitem 2
+   *     </ActionMenu.Item>
+   *   </ActionMenu.SubContent>
+   * </ActionMenu.Sub>
+   * ```
+   */
+  SubContent: typeof ActionMenuSubContent;
+}
+
+const ActionMenuRoot = ({
+  children,
+  open: openProp,
+  defaultOpen = false,
+  onOpenChange,
+}: ActionMenuProps) => {
+  const triggerRef = useRef<HTMLButtonElement>(null);
+
+  const [open = false, setOpen] = useControllableState({
+    value: openProp,
+    defaultValue: defaultOpen,
+    onChange: onOpenChange,
+  });
+
+  return (
+    <ActionMenuProvider
+      triggerId={useId()}
+      triggerRef={triggerRef}
+      contentId={useId()}
+      open={open}
+      onOpenChange={setOpen}
+      onOpenToggle={() => setOpen((prevOpen) => !prevOpen)}
+    >
+      <Menu open={open} onOpenChange={setOpen} modal={false}>
```

Comment:
The issue stemmed from trigger ending up behind the "dimsissablelayer" when open. That is now fixed by adding "pointer-events: auto" when open to the trigger. 
https://github.com/navikt/aksel/pull/3081/commits/f4a54f6bff3eeda02d340f8bfd7f4f3f054ae070

Could also then remove some other [focus-logic](https://github.com/navikt/aksel/pull/3081/commits/536ecf9be958e925644748133cf22907b67f83a4) since we always focus the trigger